strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -638,6 +638,11 @@ def prepare_gaussian_state(self, r, V, modes):
                 "Shape of covariance matrix must be [2N, 2N], where N is the number of modes."
             )
 
+        # Include these lines to accomodate out of order modes, e.g.[1,0]
```

Comment:
clever, so this makes sure things work if `modes=[2,1]`

Reply:
As is the case in one of the tests. I did not consider users giving things out of order :)